content/millennium/dstu2.md

```diff
@@ -329,9 +329,29 @@ parameter:
 $ curl -i -H "Accept: application/json+fhir" "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationOrder?patient=2744010&status=active"
 </pre>
 
-In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status` 
+In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status`
  are passed in the query string.
 
+ ### Resource Identity
+
+ Please note that no ids or identifiers in Millennium EHR are intended to be used outside of the context of their complete URL. A complete URL comprises of [service root url](http://hl7.org/fhir/dstu2/http.html#general), the resource, and parameters (if any).
+
+ For example, one must take into account the entire url and not simply the id or resource + id:
+
+     https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person/1316024
```

Comment:
To keep consistency across examples, let's use Patient for the by-id example, as well.